ts/components/conversation/header/ConversationHeaderSelectionOverlay.tsx

```diff
@@ -33,6 +35,25 @@ export const SelectionOverlay = () => {
   const isPublic = useSelectedIsPublic();
   const dispatch = useDispatch();
 
+  useKey('Delete', event => {
```

Comment:
why do we need the useKey here is we listen for key down on the Session Convo? What am I missing?

Reply:
because as soon as you select a message, the focus is token by the selection overlay so those actions are not doing anything.

Reply:
Okay. Might be worth leaving a comment to say that is the plan in future?